src/global_mapper/global_mapper_params.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 Massachusetts Institute of Technology
+#pragma once
+
+namespace global_mapper {
+class GlobalMapperParams {
+ public:
+  GlobalMapperParams();
+
+  double voxel_xyz0_[3];
+  double voxel_xyz1_[3];
+  double voxel_meters_per_pixel_[3];
+  double voxel_init_value_;
+  double voxel_min_range_;
+  double voxel_max_range_;
+  double voxel_min_z_abs_;
+  double voxel_max_z_abs_;
+  bool voxel_use_rel_cropping_;
+  double voxel_min_z_rel_;
+  double voxel_max_z_rel_;
```

Comment:
I think it's mainly an issue of communicating intent (since yes, I agree there isn't really a functional difference between classes and structs). When you declare a struct, the intent of using the struct as POD is pretty clear (which should be the case for params I think).